types/uppercamelcase/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for uppercamelcase
+// Type definitions for uppercamelcase 3.0
 // Project: https://github.com/samverschueren/uppercamelcase
 // Definitions by: York Yao <https://github.com/plantain-00>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function upperCamelCase(...args: string[]): string;
 export = upperCamelCase;
-declare namespace upperCamelCase { }
```

Comment:
Why do you want to remove the namespace?
If the namespace is removed, the `import * as upperCamelCase from "uppercamelcase";` will not work.
And the `import upperCamelCase from "uppercamelcase";` will be compiled into `const uppercamelcase_1 = require("uppercamelcase");uppercamelcase_1.uppercamelcase()`, that is not the usage of uppercamelcase.